### PR TITLE
add fizz-buzz exercise

### PR DESCRIPTION
**Pull request by marianfoo**
I have seen this exercise before and just created PR.
Please give feedback what can be improved.

### CODE REVIEW

File: exercises/practice/fizz-buzz/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "blurb": "Return 'Fizz', 'Buzz' or 'FizzBuzz' for numbers divisible by 3 or 5 but not both.",
+  "authors": ["marianfoo"],
+  "contributors": [
+  ],
+  "files": {
+    "solution": ["zcl_fizz_buzz.clas.clas.abap"],
+    "test": ["zcl_fizz_buzz.clas.clas.testclasses.abap"],
+    "example": [".meta/zcl_fizz_buzz.clas.clas.abap"]
```
Comment (g-back) [inline exercises/practice/fizz-buzz/.meta/config.json:9]: looks like there's one `clas` to many in each here:
```suggestion
    "solution": ["zcl_fizz_buzz.clas.abap"],
    "test": ["zcl_fizz_buzz.clas.testclasses.abap"],
    "example": [".meta/zcl_fizz_buzz.clas.abap"]
```

**Comment by marianfoo** [comment]
@g-back correct, you're right, I have fixed it

File: exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.abap [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class ZCL_FIZZ_BUZZ definition
+  public
+  final
+  create public .
+
+public section.
```
Comment (larshp) [inline exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.abap:6]: please pretty print this class, keywords upper case

Comment (larshp) [reply exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.abap:6]: plus the testclasses

File: exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.testclasses.abap [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+class ltcl_fizz_buzz implementation.
+
+  METHOD setup.
+    cut = NEW ZCL_FIZZ_BUZZ( ).
+  ENDMETHOD.
+
+  method test_fizz_Buzz.
+  data result TYPE string_tab11.
```
Comment (larshp) [inline exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.testclasses.abap:19]: need testcases here instead of the implementation

**Comment by larshp** [comment]
perhaps do https://github.com/exercism/problem-specifications/tree/main/exercises/raindrops instead?

**Comment by marianfoo** [comment]
> perhaps do https://github.com/exercism/problem-specifications/tree/main/exercises/raindrops instead?

Should only these exercises be used? I have now made up my own puzzle that is not present in the list.

File: exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.abap (hunk 2 above)
Comment (marianfoo) [reply exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.abap]: done

File: exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.testclasses.abap (hunk 3 above)
Comment (marianfoo) [reply exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.testclasses.abap]: The 'implementation' is there to compare the string tables with the correct solution.
Make more sense to me than creating the table manually

**Comment by larshp** [comment]
I think anything goes :)

**Comment by marianfoo** [comment]
using exercise raindrops instead of fizz buzz 
https://github.com/exercism/problem-specifications/blob/main/exercises/raindrops/description.md

**Pull request closed without merging**